src/hardware/innovation.cpp

```diff
@@ -21,10 +21,12 @@
 
 #include "control.h"
 #include "pic.h"
+#include "support.h"
 
-// Innovation Settings
-// -------------------
-constexpr uint16_t SAMPLES_PER_BUFFER = 2048;
+// Constants
+// ---------
+constexpr auto idle_after_ms = 200;
+constexpr auto amplification = 6;
```

Comment:
6 what? Shouldn't we use decibels instead?

Reply:
Dropped it here; now just doubling the sample value at the point of return and asserting that it's in-range.